forge_theme/base.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>{% if page_title %}{{ page_title }} - {% endif %}{{ site_name }}</title>
+
+    <link rel="apple-touch-icon" sizes="180x180" href="http://forge.mineformers.de/images/favicons/apple-touch-icon.png">
```

Comment:
Whoops... forgot to change this from my old design. Technically it'd be best if we had a CDN to host all this stuff on, but that might only happen some time later down the road.